src/components/SearchPage/SearchPage.jsx

```diff
@@ -1,39 +1,53 @@
 import React, { useState } from 'react';
-import axios from 'axios';
 import SearchBar from '../SearchBar/SearchBar';
 import ImageList from '../ImageList/ImageList';
 import './SearchPage.css';
-
-const API_KEY = process.env.REACT_APP_ACCESS_KEY;
+import fetchImages from '../../utils/index';
 
 const SearchPage = () => {
   const [searchText, setSearchTerm] = useState('');
-  const [image, setImage] = useState(null);
+  const [image, setImage] = useState();
   const [isLoaded, setIsLoaded] = useState(false);
+  const [isNext, setIsNext] = useState(false);
+  const [nextPageIndex, setNextPageIndex] = useState(1);
```

Comment:
Can you please explain why `useState()` is sent "false" in one statement and then "1" in the other?

Reply:
They are 2 different states, IsNext is used to know when show more button is pressed.
NextPageIndex is used to know what page to query the Api, and the default value is 1

Reply:
Ah ok, it's being imported form `react` when I checked out the code it made more sense when looking at the whole file.